main.sh

```diff
@@ -107,40 +99,40 @@ function xml_dump {
 
 function all_sch { # arg($1:group)
     # is loaded by default so data must be loaded
-    data=($(sed 's/://g' $SCHEDULE))
```

Comment:
hmm... i didn't realized alias not taking arguments. any ways you can always use function.
```bash
function batti { /usr/local/bin/2utf8 $@ | sed 's/ Group [^1]/\n&/'; }
```
Yes, 80×24 might be ignored now days, but its the still the default configuration when evey you open `gnome-terminal`, `xfce4-terminal` or `konsole`..etc. Resizing or scrolling after command execution :-1:  will be lame.

Its unlikely people change the configuration.. for this. So it worth staying in 80×24.